.github/workflows/deploy-main.yml

```
@@ -99,33 +116,40 @@ jobs:
+          tags: |
+            ${{ env.IMAGE_NAME }}
+            ${{ env.IMAGE_LATEST }}
 
-  deploy-ec2:
+  deploy-miniserver:
     needs: build-and-push
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, linux, x64, fairplay, miniserver]
+    env:
+      MINI_SERVER_DEPLOY_PATH: ${{ secrets.MINI_SERVER_DEPLOY_PATH || '/home/song/Desktop/fairplay-deploy' }}
+      FAIRPLAY_BACKEND_IMAGE: ${{ env.IMAGE_NAME }}
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)